hoot-core/src/main/cpp/hoot/core/conflate/merging/SmallWayMerger.h

```diff
@@ -76,12 +76,18 @@ class SmallWayMerger : public OsmMapOperation
    */
   static void mergeWays(boost::shared_ptr<OsmMap> map, Meters threshold);
 
-  virtual QString getDescription() const { return "Merges small ways"; }
+  virtual QString getInitStatusMessage() { return "Merging very small ways..."; }
+
+  virtual QString getCompletedStatusMessage()
+  { return "Merged " + QString::number(_numMerged) + " very small ways"; }
+
+  virtual QString getDescription() const { return "Merges very small ways"; }
 
 protected:
 
   boost::shared_ptr<OsmMap> _map;
 
+  int _numMerged;
```

Comment:
In a lot of these classes based on `OsmMapOperation` you have `_num<something>` whether it is splitting or merging or counting.  Would it make sense to add the counter as `_numAffected` to the base `OsmMapOperation` class and eliminate it from having to be duplicated?

Reply:
Yeah, I think that's a good idea...cleaner.  That var can just be ignore in some of the other classes that don't have that type of metric.  Occasionally an op will calculate multiple things, so maybe a list of vars.